### PR TITLE
Feature/2176 long relations hide arcs

### PR DESCRIPTION
**Pull request by jcklie**
**What's in the PR**
- Add vdoc to render method
- Fix brat errors
- Add option to hide/show arcs and render a fallback
- Enable string recommender for relations by default

**How to test manually**
* Create base and relation layer, set `Show arcs` to false
* Annotate things with relations spanning several sentences

**Automatic testing**
* [ ] PR includes unit tests

**Documentation**
* [ ] PR updates documentation

### CODE REVIEW

File: inception/inception-imls-stringmatch/src/main/java/de/tudarmstadt/ukp/inception/recommendation/imls/stringmatch/config/StringMatchingRecommenderAutoConfiguration.java [hunk 1]
```diff
@@ -82,7 +81,6 @@ public StringMatchingRecommenderFactory stringMatchingRecommenderFactory(
     }
 
     @Bean
-    @ConditionalOnProperty(prefix = "recommender.string-matching.relation", name = "enabled", havingValue = "true")
```
Comment (reckart) [inline inception/inception-imls-stringmatch/src/main/java/de/tudarmstadt/ukp/inception/recommendation/imls/stringmatch/config/StringMatchingRecommenderAutoConfiguration.java:85]: Why did you remove the conditional?

Comment (jcklie) [reply inception/inception-imls-stringmatch/src/main/java/de/tudarmstadt/ukp/inception/recommendation/imls/stringmatch/config/StringMatchingRecommenderAutoConfiguration.java:85]: I think its time for them to be enabled by default in 21

Comment (reckart) [reply inception/inception-imls-stringmatch/src/main/java/de/tudarmstadt/ukp/inception/recommendation/imls/stringmatch/config/StringMatchingRecommenderAutoConfiguration.java:85]: I think that deserves a separate PR, no? Also the documentation probably needs to be updated then to remove the "experimental" notice there.

Comment (jcklie) [reply inception/inception-imls-stringmatch/src/main/java/de/tudarmstadt/ukp/inception/recommendation/imls/stringmatch/config/StringMatchingRecommenderAutoConfiguration.java:85]: I can do that

Comment (reckart) [reply inception/inception-imls-stringmatch/src/main/java/de/tudarmstadt/ukp/inception/recommendation/imls/stringmatch/config/StringMatchingRecommenderAutoConfiguration.java:85]: I believe pulling out the change to enable the relation recommender by default is the only "blocker" for merging this PR, right?

Comment (jcklie) [reply inception/inception-imls-stringmatch/src/main/java/de/tudarmstadt/ukp/inception/recommendation/imls/stringmatch/config/StringMatchingRecommenderAutoConfiguration.java:85]: I wanted to wait for feedback from the guy, but yes, that is the only blocker if you think its ready to be merged otherwise.

Comment (jcklie) [reply inception/inception-imls-stringmatch/src/main/java/de/tudarmstadt/ukp/inception/recommendation/imls/stringmatch/config/StringMatchingRecommenderAutoConfiguration.java]: I undid it
